velox/functions/sparksql/tests/SparkCastExprTest.cpp

```diff
@@ -192,6 +192,68 @@ TEST_F(SparkCastExprTest, decimalToIntegral) {
   testDecimalToIntegralCasts<int8_t>();
 }
 
+TEST_F(SparkCastExprTest, decimalToVarchar) {
+  auto expect64 = [&](const std::vector<std::optional<int64_t>>& values,
```

Comment:
Could you use like other tests to test?
```
    testCast(
        shortFlat,
        makeNullableFlatVector<T>(
            {-3,
             -2 /*-2.6 truncated to -2*/,
             -2 /*-2.3 truncated to -2*/,
             -2,
             -1,
             0,
             55,
             57 /*57.49 truncated to 57*/,
             57 /*57.55 truncated to 57*/,
             69,
             72,
             std::nullopt}));
```

Reply:
Using the lambda helper function can reduce the same comparing codes

Reply:
rename to testShortDecimal

Reply:
done